ArduPlane/soaring.cpp

```diff
@@ -66,156 +66,118 @@ void Plane::update_soaring() {
         // Update thermal estimate and check for switch back to AUTO
         g2.soaring_controller.update_thermalling();  // Update estimate
 
+        // Thermalling is done in a home-relative coordinate system, so we need home to be set.
+        Vector3f position;
+        if (!ahrs.get_relative_position_NED_home(position)) {
+            return;
+        }
+
+        // Check distance to home against MAX_RADIUS.
+        if (g2.soaring_controller.max_radius >= 0 &&
```

Comment:
zero here is an interesting value -  Is it worth saying zero-disables instead?

Reply:
Left unchanged for now.